### ISSUE TITLE
time cost and CPU

### ISSUE THREAD

**Issue by woshi0655** [hwc2021/GSAT#6]
Hi, 
     Why only graphShort step has CPU parameter, no CPU parameter for other step? And graphLong graphCorrection step seems cost  a lot of time, is that normal ?
     Is it possible to change the kmer when run the graphShort pipeline with spades

**Comment by awesomedeer** [comment]
True.. One-step command is good although I can't see other parameters to adjust each step.

**Comment by hwc2021** [comment]
Hi!

The current version of GSAT does not support multithreading, which would be improved in the next version.

For a og.filtered.gfa file < 1Mb, it usually costs < 1 day in graphLong step and < 2 day in graphCorrection step. The og.filtered.gfa should be checked and edited in Bandge if it shows a abnormal size, as I answered for another issue (https://github.com/hwc2021/GSAT/issues/5#issuecomment-1751885293).

The program automatically determines the appropriate kmer settings according to the sequencing reads data.
